linkerd/app/integration/src/proxy.rs

```diff
@@ -244,17 +244,16 @@ async fn run(proxy: Proxy, mut env: TestEnv, random_ports: bool) -> Listening {
         Some(identity.addr)
     } else {
         env.put(app::env::ENV_IDENTITY_DISABLED, "test".to_owned());
-        env.put(app::env::ENV_TAP_DISABLED, "test".to_owned());
         None
     };
 
     // If identity is enabled but the test is not concerned with tap, ensure
     // there is a tap service name set
-    if !env.contains_key(app::env::ENV_TAP_DISABLED)
-        && !env.contains_key(app::env::ENV_TAP_SVC_NAME)
-    {
-        env.put(app::env::ENV_TAP_SVC_NAME, "test-identity".to_owned())
-    }
+    // if !env.contains_key(app::env::ENV_TAP_DISABLED)
+    //     && !env.contains_key(app::env::ENV_TAP_SVC_NAME)
+    // {
+    //     env.put(app::env::ENV_TAP_SVC_NAME, "test-identity".to_owned())
+    // }
```

Comment:
Can we remove this (and the comment above)?

Reply:
Whoops yep meant to remove that.